build/lib/src/builder/post_process_build_step.dart

```diff
@@ -9,25 +9,30 @@ import 'package:async/async.dart';
 import 'package:build/build.dart';
 import 'package:crypto/src/digest.dart';
 
-import '../asset_graph/graph.dart';
-import '../asset_graph/node.dart';
+// This is not exported to hack around a package-private constructor.
```

Comment:
Not familiar with this limitation. Can you explain further?

Reply:
What is the advantage of this versus just not exporting the constructor and making it public?

Reply:
We export the type because users need to use it to implement a `PostProcessBuilder` but we don't want users outside the package to be able to instantiate one.